README.md

```diff
@@ -53,6 +53,67 @@ The end user chooses which hydrator to use. The default one should return domain
 
 The request builder will build request with multipart streams when necessary. 
 
+### Domain objects as parameters
+
+If the API requires lots of parameters for a specific endpoint it could be tempting 
+to create a domain object and pass it as an argument to that endpoint. 
+
+```php
+public function create(string $username, Tweet $model) {
+    // send the Tweet to Fake Twitter API
+    // ...
+}
+
+$model = new Tweet();
+$model->setMessage('foobar');
+$model->addHashTag('stuff');
+$model->addHashTag('test');
+$model->setLocation('Stockhom/Sweden');
+// ...
+$api->create('foobar', $model);
+```
+
+This approach, however, is not preferred as the created Tweet object is an unnecessary 
+overhead. It could also conflict with the application developers' Tweet object. 
```

Comment:
That is correct. If it is one developer I would write: "application developer's ". Since it is plural I need to write "application developers' " instead of "application developers's ".